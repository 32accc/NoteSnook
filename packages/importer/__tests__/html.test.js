const html = require("../providers/html");

test("convert evernote note enex file", () => {
  let file = {
	  data:"<p>Hello world</p>",
	  fileName:"world.html",
	  createdAt:0,
	  modifiedAt:0
  }

  expect(html.convert([file])).toMatchSnapshot("html-file");
});
