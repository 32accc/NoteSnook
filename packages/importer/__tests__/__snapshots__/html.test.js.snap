// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`convert evernote note enex file: html-file 1`] = `
Array [
  Object {
    "content": Object {
      "data": "<p>Hello world</p>",
      "type": "html",
    },
    "dateCreated": 0,
    "dateEdited": 0,
    "title": "world",
  },
]
`;
